crates/net/eth-wire/src/types/broadcast.rs

```diff
@@ -304,16 +304,37 @@ impl Arbitrary for NewPooledTransactionHashes68 {
     type Parameters = ();
 
     fn arbitrary_with(_args: ()) -> Self::Strategy {
-        (any::<usize>(), any::<u8>(), any::<usize>(), any::<B256>())
-            .prop_map(|(size, type_val, size_val, hashes_val)| NewPooledTransactionHashes68 {
-                types: vec![type_val; size],
-                sizes: vec![size_val; size],
-                hashes: vec![hashes_val; size],
+        (vec(any::<TxType>(), 1..100), vec(any::<usize>(), 1..100), vec(any::<B256>(), 1..100))
+            .prop_map(|(types, sizes, hashes)| {
+                // Ensure the vectors have the same length
+                let min_length = *[types.len(), sizes.len(), hashes.len()].iter().min().unwrap();
```

Comment:
I wonder if this can improved by using a random length value for all the vec strategies instead?